apps/go-mod-validator/src/deps.ts

```diff
@@ -0,0 +1,47 @@
+import { execSync } from "child_process";
+import * as core from "@actions/core";
+
+function getGoModFiles(goModDir: string): string[] {
+  let output = "";
+
+  try {
+    output = execSync(`find ${goModDir} -type f -name 'go.mod'`, {
+      encoding: "utf-8",
+    });
+  } catch (error) {
+    throw new Error(`failed to get go.mod files: ${error}`);
+  }
+
+  if (output.length == 0) {
```

Comment:
```suggestion
  if (output.length === 0) {
```

It's best to use strict equality checks